.github/workflows/pulp_images.yml

```diff
@@ -90,7 +91,7 @@ jobs:
           podman version
           buildah version
           podman build --format docker --file images/Containerfile.core.base --tag pulp/base:${TEMP_BASE_TAG} .
-          podman build --format docker --file images/pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:${TEMP_BASE_TAG} --build-arg FROM_TAG=${TEMP_BASE_TAG} .
+          podman build --format docker --file images/pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:${TEMP_BASE_TAG} --build-arg FROM_TAG=${TEMP_BASE_TAG} --build-arg PULP_FILE_VERSION=${PULP_FILE_VERSION} .
```

Comment:
In this base-ci-image, noplugins are installed at all. I don't think this change is necessary or will even change a thing.

Reply:
Thank you! Updated to put PULP_FILE_VERSION on the correct image layer.